agent/config/testdata/TestRuntimeConfig_Sanitize.golden

```diff
@@ -462,6 +462,7 @@
         "DisableHostname": false,
         "DogstatsdAddr": "",
         "DogstatsdTags": [],
+        "ExtraSinks": [],
```

Comment:
Why do we need this? Have we added a new config value somewhere?

Reply:
yep! https://github.com/hashicorp/consul/pull/17162/files#r1180516809 - this comment. New field in the Telemetry package's TelemetryConfig struct.

Reply:
Its not a config value that is settable by the user directly. A bit like `TelemetryConfig.PrometheusOpts`